pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/OperatingSystem.java

```diff
@@ -0,0 +1,384 @@
+/*
```

Comment:
Nooo.  Don't do it like that.  The gradle developers I have been working with on another project recommend:

+import org.apache.tools.ant.taskdefs.condition.Os

You can look at my PR for examples on use.  I reached a stopping point at my pay job today and was going to pick this up again in the morn.  I'll take a look at what you did here a bit closer.